www/js/controllers/helloWorldController.js

```diff
@@ -1,3 +1,15 @@
-export default [ '$scope', function( $scope ) {
-		$scope.helloWorld = "Hello World Controller";
-}];
+export default class {
+
+    static get $inject() {
```

Comment:
Using an ES6 Class, we can define a static $inject() method that returns an array of the named injectables to be passed to the constructor.